app/repository/hbase/legalunit/LegalUnitColumns.scala

```diff
@@ -0,0 +1,9 @@
+package repository.hbase.legalunit
+
+object LegalUnitColumns {
+  val uBRN = "UBRN"
+  val UBRNref = "UBRNref"
+  val ern = "ern"
+  val entref = "entref"
+
```

Comment:
Need to add the other columns even those that are optional (as declared in LegalUnitRowMapper.scala)

Reply:
There is no such thing as UBRNREF. Enterprises have two ids, the new id from SBR and the IDBR id. Legal units are new so only have UBRNs thus remove the UBRNref val.

Reply:
Address is being added to legal units (currently in BI there is only a postcode) so please include the address parts and postcode in the columns. Mandatory is address1 and postcode as per the other unit types.